### ISSUE TITLE
Painkiller issue

### ISSUE THREAD

**Issue by Lolcsi** [EpochModTeam/DayZ-Epoch#1788]
I have overburdened and use painkiller but nothing happened.

**Comment by AirwavesMan** [comment]
@ebaydayz 
in 1051 when you were overburdened you could cure that effect with painkillers. I think it was not intentional but it worked very well. I think thats @Lolcsi  is a bit confused

**Comment by ebaydayz** [comment]
My bad, you are correct. 1051 did reset the tired accumulator when you took painkillers:
https://github.com/EpochModTeam/DayZ-Epoch/blob/Release_1.0.5.1/SQF/dayz_code/medical/publicEH/medPainkiller.sqf#L10

Added with dd2a21a

**Comment by Lolcsi** [comment]
Yes its true. Just copy R3F_TIRED_Accumulator = 0; in the medPainkiller.sqf. It's working.

**Comment by Lolcsi** [comment]
I test overburdened. I think unconscious better than force walk. What you think about this?

**Comment by oiad** [comment]
When I'm lifting/moving heavy stuff in real life I walk quite a bit slower, never once passed out, it is a lot more realistic with the force walk which is why it was changed from memory

**Comment by icomrade** [comment]
I have to agree with @oiad, slow walking is better, although the speed limit is a bit aesthetically unpleasing. Besides there's only a slight boost in speed slow walking rather than rolling
